iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java

```diff
@@ -38,6 +38,9 @@ public String getName() {
         return name != null ? name : id;
     }
 
+    @Override
+    public abstract NetworkImpl getNetwork();
```

Comment:
Is it really useful?

Reply:
Probably not except if we need a NetworkImpl to notify. @tadam50 could you confirm this?

Reply:
I need NetworkImpl in order to notify changes on setProperty method